### ISSUE TITLE
Suggested training configurations for smaller batch sizes?

### ISSUE THREAD

**Issue by NaomiEX** [linxuewu/Sparse4D#52]
I was wondering whether you have tested with different batch sizes, in particular do you have any training config suggestions if I'm using total batch size of 8 or 16 or 32?

**Comment by linxuewu** [comment]
When the batch size is halved, it is recommended to multiply the learning rate by 2/3, while keeping other configurations unchanged.

**Comment by NaomiEX** [comment]
I see, thank you!
